CMakeLists.txt

```diff
@@ -306,7 +306,7 @@ if(HIP_FOUND AND Libva_FOUND AND Libdrm_amdgpu_FOUND)
   # (AMD build of libva 2.16) should be used if distro's libva is too old
   set(ROCDECODE_DEBIAN_RUNTIME_PACKAGE_LIST  "hip-runtime-amd, libva-drm2 (>= 2.16.0) | libva-amdgpu-drm2, mesa-amdgpu-va-drivers")
   # Unfortunately RPM has a mix of naming schemes; RHEL has "libva", SLE has "libva2-drm2"
-  set(ROCDECODE_RPM_RUNTIME_PACKAGE_LIST  "hip-runtime-amd, (libva >= 2.16.0 or libva-drm2 >= 2.16.0 or libva-amdgpu), mesa-amdgpu-va-drivers")
+  set(ROCDECODE_RPM_RUNTIME_PACKAGE_LIST  "hip-runtime-amd, (libva or libva-drm2), mesa-amdgpu-va-drivers")
```

Comment:
Why are you dropping the 2.16.0 requirement? The Mesa expects the minimum version of 2.16 in the runtime; otherwise, it throws an error.

Reply:
For RHEL 8, the requirement is lower now as per the commit description due to change in support policy, so I just dropped it since RHEL 9/10 and SLE 15 all provide > 2.16 now.

If you would like to detect OS and put the requirement, feel free, but I don't think it's necessary anymore.